views/components/Sidebar/NavItem.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { ListItem, ListItemIcon, ListItemText } from '@mui/material'
+import SidebarLink from './SidebarLink'
+
+const NavItem = ({ to, Icon, label }) => {
+  return (
+    <ListItem component={SidebarLink} to={to} sx={{ width: 228 }}>
```

Comment:
do we need explicit width values here? Should this not take up the full width of the side nav?

Reply:
The designs have a width set for this element

Reply:
There is a width in the design, but there's a width on every item in the design. In terms of implementation, I think we want to specify the sidebar width and padding, then have this be a regular block element with unspecified width.

![Screenshot 2023-11-06 at 1 32 56 PM](https://github.com/AMP-SCZ/dpdash/assets/2905145/5396714d-283c-4073-a492-c7ce23a16098)